examples/using-cron-jobs/main_test.go

```diff
@@ -8,6 +8,7 @@ import (
 )
 
 func Test_UserPurgeCron(t *testing.T) {
+	// TODO find a way to do not have to wait for more than a minute
```

Comment:
This one may need a separate PR.

We have to use a wait to speed up the clock.

There are multiple lib for that, but it requires some research and tests

Reply:
it needs to be handled in a separate PR

Reply:
I opened

- https://github.com/gofr-dev/gofr/issues/916